src/components/Modals/AddContactModal.jsx

```diff
@@ -14,45 +14,35 @@ import InputAdornment from '@mui/material/InputAdornment';
 import TextField from '@mui/material/TextField';
 // Utility Imports
 import { runNotification } from '@utils';
-// Context Imports
-import { UserListContext } from '@contexts';
-// Model Imports
-import { createUser } from '../../model-helpers/User';
 // Component Imports
 import { FormSection } from '../Form';
 
 /**
- * AddClientModal Component - Component that allows users to add other user's
+ * AddContactModal Component - Component that allows users to add other user's
```

Comment:
Just noticed this now, but I think we could move the JSDoc here to above `AddContactModal` instead of `renderWebId`. In this order, JSDoc mistaken `renderWebId` for `AddContactModal`.

<img width="825" alt="Screen Shot 2023-08-10 at 16 11 50" src="https://github.com/codeforpdx/PASS/assets/14917816/0dfa8357-c57b-4092-a517-28730936806c">

Could add a separate JSDoc comment for `renderWebId`.

Reply:
fixed